app/src/main/java/ar/com/wolox/android/login/ui/training/LoginPresenter.java

```diff
@@ -0,0 +1,72 @@
+package ar.com.wolox.android.login.ui.training;
+
+import ar.com.wolox.android.login.utils.UserSession;
+import ar.com.wolox.wolmo.core.presenter.BasePresenter;
+
+import javax.inject.Inject;
+
+public class LoginPresenter extends BasePresenter<ILoginView> {
+
+    // Constants
+    public static final String TAG = "LoginPresenter";
+    public static final int NUMBER_MAX = 1000;
+    public static final int NUMBER_MIN = 200;
+
+    // Variables
+    private UserSession mUserSession;
+
+    // Constructor
+    @Inject
+    public LoginPresenter(UserSession userSession) {
+        mUserSession = userSession;
+    }
+
+    public void login(String email, String password) {
+        Boolean validEmail = this.validateEmailField(email);
```

Comment:
You don't need to use `this` to call inner methods :smile:

Reply:
TODO this will be solved in the next PR.